src/transformers/commands/add_new_model.py

```diff
@@ -102,6 +102,11 @@ def run(self):
 
         model_dir = f"{path_to_transformer_root}/src/transformers/models/{lowercase_model_name}"
         os.makedirs(model_dir, exist_ok=True)
+        os.makedirs(f"{path_to_transformer_root}/tests/{lowercase_model_name}", exist_ok=True)
+
+        # Tests require submodules as they have parent imports
+        with open(f"{path_to_transformer_root}/tests/{lowercase_model_name}/__init__.py", 'w') as fp:
+            pass
```

Comment:
At some point, we should rewrite that file to work on Windows :-)

Reply:
Absolutely, we should